### PR TITLE
feat: Translation support

### PR DESCRIPTION
**Pull request by luismanson**
Hi, this is a basic i18next implementation, language has been set to English as default/fallback. 
Also  added react Suspense and Mantine LoadingOverlay.

PS: I think the copy/copied tootltip is not working

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
-import React from 'react';
+import { React, Suspense } from 'react';
```
Comment (bjarneo) [inline src/index.js:1]: I don't think you can import react this way, right?
```js
import React, { Suspense } from 'react';
```

**Comment by bjarneo** [comment]
Looks good to me. Just one comment about the import. That is it. Great work!

**Comment by bjarneo** [comment]
> PS: I think the copy/copied tootltip is not working

hm, yes, seems to be something fishy on mobile here. Works in Chrome for me on desktop at least

File: src/index.js (hunk 1 above)
Comment (luismanson) [reply src/index.js]: Sorry for that, I need to read a bit more....it should be fixed now.

**Comment by bjarneo** [comment]
Seems that these are missing from the package.json:
```
import i18next from 'i18next';
import { initReactI18next } from 'react-i18next';
import HttpApi from 'i18next-http-backend';
```

File: src/index.js (hunk 1 above)
Comment (bjarneo) [reply src/index.js]: Nothing to say sorry about. And, that is awesome 🚀

**Comment by luismanson** [comment]
I forgot to take care of package.json, I did not merge it, since after pulling the code npm refused to run "client-dev". 
Had to install "@emotion/cache, @emotion/react and webpack" until it worked :confused: 

How would you handle this? I could just revert my file and add only the missing packages related to this PR, not sure if doing this would introduce new issues.

**Comment by bjarneo** [comment]
I can have a look at this for you. For my part I did not have to do what you needed to. Just pull the code, run npm install, and then it should work. But let me test the PR locally.

**Comment by bjarneo** [comment]
https://github.com/HemmeligOrg/Hemmelig.app/pull/83

**Pull request closed without merging**